kotlin/android/app/build.gradle.kts

```diff
@@ -154,11 +154,8 @@ android {
 }
 
 dependencies {
-    val coreVersion = "1.12.0"
-    val navVersion = "2.7.6"
-
     // AndroidX
-    implementation("androidx.core:core-ktx:$coreVersion")
+    implementation("androidx.core:core-ktx:1.12.0")
```

Comment:
So did dependabot not know how to handle deps mentioned in variables like that?

Reply:
Yeah I don't think it figured it out in this case.